libc/test/src/time/CMakeLists.txt

```diff
@@ -136,6 +138,7 @@ add_libc_test(
     clock_test.cpp
   DEPENDS
     libc.include.time
-    libc.src.time.clock
+    libc.src.__support.CPP.limits
```

Comment:
it's unclear to me when we need to add these DEPENDS vs not.  I didn't for many changes, and the build succeeds. Is that expected? Do we have some convention here that perhaps I'm unaware of?

Reply:
DEPENDS primarily tells cmake which object files need to be linked. For header files it's technically unnecessary, though we should still do it to make sure that cmake properly rebuilds targets that depend on those header files.

Reply:
If I change this line in particular s/limits/asdfasdfasdf/ literally nothing happens to the build...that doesn't instill confidence that this change is even doing anything...

Reply:
Is that a feature or a bug?

Reply:
It's a feature, so that we skip the tests if its entrypoints are not enabled instead of erroring out, or something like that.